product-backend/src/main/java/org/kainos/ea/validator/RegisterValidator.java

```diff
@@ -0,0 +1,50 @@
+package org.kainos.ea.validator;
+
+import org.kainos.ea.cli.RequestEmployee;
+
+public class RegisterValidator {
```

Comment:
Usually you try to avoid adding a comments unless something is very complex

Reply:
removed unnecessarily comments